guides/release-process.md

```diff
@@ -77,25 +77,22 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
 
 2. If there is a new [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink/releases) version, update the corresponding dependency in [`packages/example`](../packages/example) to that new version.
 
-3. Use the `move-binaries` script to move the binaries for `<commit sha>` from `beta` to the `desktop` folder for `<new target version>`. This also purges the cloudflare cache for this version.
+3. Use the `prepare-release-artifacts` script to prepare the latest commit to a stable release. When you run this script, the following happens:
+    * the binaries for `<commit sha>` are moved from `beta` to the `desktop` folder for `<new target version>` in S3
+    * the Cloudflare cache for this version is purged
+    * the pre-prod `cypress.tgz` NPM package is converted to a stable NPM package ready for release
 
     ```shell
-    yarn move-binaries --sha <commit sha> --version <new target version>
+    yarn prepare-release-artifacts --sha <commit sha> --version <new target version>
```

Comment:
Since this run's on the releaser's local machine, should there be a requirements that this must run on a Mac or Linux machine to ensure line-endings are not corrupted?

Reply:
The script will always download the `linux-x64` .tgz, but you raise a good point, possibly the `stable: true` write will mess with the line endings. I'll test this out and see.

Reply:
In the process of testing this, I discovered that this script relies on behavior of `tar` that is not present on Windows, so it fails anyways. Since only internal contributors use this script and we're all on MacOS and Linux, I went ahead and added a guard like you suggested [bd305d1](https://github.com/cypress-io/cypress/pull/20296/commits/bd305d1e90d4d86453bffeff1a77758edc03cff6)